network-api/networkapi/wagtailpages/pagemodels/customblocks/blog_index_callout_box.py

```diff
@@ -0,0 +1,50 @@
+from wagtail.core import blocks
+from wagtail.snippets import blocks as snippet_blocks
+from ..customblocks.base_rich_text_options import base_rich_text_options
+from wagtailmedia.blocks import AudioChooserBlock
+from ..blog.blog_topic import BlogPageTopic
+
+
+class BlogIndexCalloutBlock(blocks.StructBlock):
+
+    title = blocks.CharBlock(
+        help_text='Heading for the callout box.'
+    )
+
+    related_topics = blocks.ListBlock(
+        snippet_blocks.SnippetChooserBlock(BlogPageTopic),
+        max_num=2,
+        help_text='Optional topics to display at the top of the callout box.'
+
+        )
```

Comment:
**Nitpick** Should be outdented one level

Reply:
Updated